surfnperf.js

```diff
@@ -356,6 +356,59 @@
     return this.duration('navigationStart', 'loadEventEnd');
   };
 
+  /**
+   * Time to first paint (browser-reported time for the first pixels to be rendered on the page)
+   *
+   * @returns {integer} time in ms
+   * @memberOf SurfNPerf
+   */
+  SNPProto.getFirstPaint = function(options) {
+    if(this.chromeLoadTimes()) {
+      this._data.marks.chromePaint = this.chromeLoadTimes().firstPaintTime * 1000;
+      return this.duration('navigationStart', 'chromePaint', options);
```

Comment:
the 2 lines in this `if(this.chromeLoadTimes()) {` block can be replaced with this one:
```js
return this._roundedDuration(this.getTimingMark('navigationStart', 'DOM'), this.chromeLoadTimes().firstPaintTime * 1000, options);
```